tests/python/unittest/test_random.py

```diff
@@ -554,7 +554,8 @@ def compute_expected_prob():
     mx.test_utils.assert_almost_equal(exp_cnt_sampled.asnumpy(), exp_cnt[sampled_classes].asnumpy(), rtol=1e-1, atol=1e-2)
     mx.test_utils.assert_almost_equal(exp_cnt_true.asnumpy(), exp_cnt[true_classes].asnumpy(), rtol=1e-1, atol=1e-2)
 
-@with_seed()
+# See issue #10277 for the fixed seed.
```

Comment:
Please add a link to ensure that developers are able to resolve the reference in future - e.g. when we change bug trackers

Reply:
I added the link.